crates/frontend/src/api.rs

```diff
@@ -248,3 +269,21 @@ pub async fn fetch_types(
         .await
         .map_err(err_handler)
 }
+
+pub async fn get_config_by_version(tenant: String, version: String) -> Result<Value, ServerFnError> {
```

Comment:
Why do you need this api to be called?
Can we not use the get snapshots list API data here itself?

Reply:
This is for single snapshot page. 
I did not want to do the parsing .

Reply:
parsing or passing the data?